lib/routes/environment.py

```diff
@@ -0,0 +1,93 @@
+"""
+Environment routes
+"""
+from fastapi import APIRouter
+
+from lib.views.environment import (
+    EnvSummary,
+    EnvCreated,
+    EnvUpdated,
+    EnvDeleted,
+    EnvPickle,
+)
+from lib.models.environment import Env
+from lib.controllers.environment import EnvController
+
+router = APIRouter(
+    prefix="/environments",
+    tags=["ENVIRONMENT"],
+    responses={
+        404: {"description": "Not found"},
+        422: {"description": "Unprocessable Entity"},
+        500: {"description": "Internal Server Error"},
+    },
+)
+
+
+@router.post("/")
+async def create_env(env: Env) -> "EnvCreated":
```

Comment:
Is there any particular reason for using quotes the annotation here? 

`"EnvCreated"` instead of `EnvCreated`

Reply:
This is a [forward reference](https://peps.python.org/pep-0563/#:~:text=no%20longer%20supported.-,Forward%20References,block%20a%20forward%20reference%2C%20too.) to handle reference issues we had in the past. I had to do it like this because AWS app runner was running in a python version bellow 3.7 so some classes/functions had problems with type hints, PEP-563 solves this as they stop to check type hint at function definition time (see [Rationale and Goals](https://peps.python.org/pep-0563/)). 

The same design was applied to every type hint just for matter of standardization. 

Recently AWS [added support for python 3.11](https://docs.aws.amazon.com/apprunner/latest/relnotes/release-2023-12-29-python-node-prebuild.html), but now we are using an ECS so 🤷‍♂️.

Anyway, since we started using containers this has no longer been an issue, we can now safely restore these hints to obj notation instead of strings. But I recommend we do it later if it is ok for you (maybe something I can ask the new joiners so they start getting familiar with the API code and how to test changes on it)

Reply:
Ok, that's valuable information.

I have no problem postponing this change, I was simply curious.